### DISCUSSION TITLE
5–9 Sept.

### DISCUSSION THREAD

**Discussion by esheehan-gsl** [NOAA-GSL/unified-graphics#86] [Progress Report]
🚴🏻 **Progress**

- Experimenting with #85 

😖 **Regressions**

None

💬 **To Discuss**

None

📆 **Next Week**

- Complete #85 
- Add brushing interaction to histograms

**Comment by ian-noaa** [comment]
🚴🏻 **Progress**

- Ian out sick. :face_with_thermometer: 

😖 **Regressions**

None

💬 **To Discuss**

- #89 

📆 **Next Week**

- #80 
- #67
